lib/cmd/common/import.js

```diff
@@ -24,7 +24,7 @@ fs.existsSync = fs.existsSync || path.existsSync;
 // main import entry point
 function imports (argv, cb) {
   var args = argv._,
-  isNGUI = ini.get('fhversion') >= 3;
+    isNGUI = ini.get('fhversion') >= 3;
```

Comment:
This is actually the correct indentation, auto-corrected by `eslint`, we have to argue whether we should avoid using `var x = 1, y = 2;` instead.